src/analytics/selectors.ts

```diff
@@ -59,8 +59,8 @@ export const getCurrentUserTraits = createSelector(
     // Don't rename these unless you have a really good reason!
     // They are used in users analytics profiles + super properties
     return {
-      accountAddress,
-      walletAddress,
+      accountAddress: mtwAddress ?? rawWalletAddress,
+      walletAddress: rawWalletAddress?.toLowerCase(),
```

Comment:
This is necessary because accountAddressSelector did not lowercase the wallet address when taking it from redux state, but walletAddressSelector does. We've been lower-casing wallet address before saving it to redux state since 11/2019, but for older users they might not be lowercase. (There's no redux migration in place for those users!) So I did this to be safe...